src/WebJobs.Script/Http/RetryProxyHandler.cs

```diff
@@ -55,7 +74,7 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 {
                     var message = attemptCount == MaxRetries
                         ? "Reached the maximum retry count for worker request proxying. Error: {exception}"
-                        : $"Unsupported exception type in {nameof(RetryProxyHandler)}. Request will not be retried. Exception: {{exception}}";
+                        : $"HTTP requests will not be retried. Excpetion in {nameof(RetryProxyHandler)}: {{exception}}.";
```

Comment:
request**s**? Isn't this a http retry handler?

```suggestion
                        : $"HTTP request will not be retried. Excpetion in {nameof(RetryProxyHandler)}: {{exception}}.";
```

Reply:
Same on line 62?

Reply:
Fixed!